app-center-services/src/main/java/org/exoplatform/appcenter/service/ApplicationCenterService.java

```diff
@@ -276,8 +294,14 @@ public Application createApplication(Application application) throws Exception {
     if (application.getPermissions() == null || application.getPermissions().isEmpty()) {
       application.setPermissions(DEFAULT_USERS_PERMISSION);
     }
-
-    return appCenterStorage.createApplication(application);
+    Application createdApplication = appCenterStorage.createApplication(application);
+    long appId  =createdApplication.getId();
+    try {
+      listenerService.broadcast(this.POST_CREATE_APPLICATION, appId, null);
+    } catch (Exception e) {
+      LOG.warn("Error broadcasting event '" + this.POST_CREATE_APPLICATION + "' using source '" + appId + "' and data " + null, e);
```

Comment:
IMO the null should be removed from the Log message :)
```suggestion
      LOG.warn("Error broadcasting event {} using source {}", this.POST_CREATE_APPLICATION, appId,  e);
```